examples/preprocessing/eog_regression.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+"""
+=======================================
+Reduce EOG artifacts through regression
+=======================================
+
+Reduce artifacts by regressing the EOG channels onto the rest of the channels
+and then subtracting the EOG signal.
+
+This is a quick example to show the most basic application of the technique.
+See the `tutorial<_tut-artifact-regression>`_ for a more thorough explanation
+that demonstrates more advanced approaches.
+
+Authors: Marijn van Vliet <w.m.vanvliet@gmail.com>
+
+License: BSD (3-clause)
+"""
+# %%
+# Do imports and load the MNE-Sample data.
+import warnings
+
+import mne
+from mne.datasets import sample
+from mne.preprocessing import EOGRegression
+from matplotlib import pyplot as plt
+
+print(__doc__)
+
+data_path = sample.data_path()
+raw_fname = data_path / 'MEG' / 'sample' / 'sample_audvis_filt-0-40_raw.fif'
+
+# Read raw data
+raw = mne.io.Raw(raw_fname, preload=True)
+events = mne.find_events(raw, 'STI 014')
+
+# Highpass filter to eliminate slow drifts
+raw.filter(0.3, None, picks='all')
+
+# %%
+# Perform regression and remove EOG
+# ---------------------------------
+
+weights = EOGRegression().fit(raw)
+raw_clean = weights.apply(raw, copy=True)
+
+# Show the filter weights in a topomap
+weights.plot()
+
+# %%
+# Before-after comparison
+# -----------------------
+# Let's compare the signal before and after cleaning with EOG regression. This
+# is best visualized by cutting epochs and plotting the evoked potential.
+
+tmin, tmax = -0.1, 0.5
+event_id = {'VisL': 3, 'VisR': 4}
```

Comment:
```suggestion
event_id = {'visual/left': 3, 'visual/right': 4}
```

This is more consistent with what other examples/tutorials do.